interface/modules/custom_modules/oe-module-weno/sql/table.sql

```diff
@@ -0,0 +1,89 @@
+#IfTable weno_pharmacy
+DROP TABLE IF EXISTS `weno_pharmacy`;
+#EndIf
+
+#IfNotTable weno_pharmacy
+CREATE TABLE `weno_pharmacy` (
+  `id` int(20) NOT NULL AUTO_INCREMENT,
+  `App` varchar(8) DEFAULT NULL,
+  `NCPDP` varchar(8) DEFAULT NULL,
+  `NCPDP_safe` varchar(7) DEFAULT NULL,
+  `Mutually_Defined_ID` varchar(10) DEFAULT NULL,
+  `Mutually_Defined_ID_safe` varchar(10) DEFAULT NULL,
+  `NPI` varchar(10) DEFAULT NULL,
+  `NPI_safe` varchar(10) DEFAULT NULL,
+  `Business_Name` varchar(255) DEFAULT NULL,
+  `Address_Line_1` varchar(255) DEFAULT NULL,
+  `Address_Line_2` varchar(255) DEFAULT NULL,
+  `City` varchar(20) DEFAULT NULL,
+  `State` varchar(20) DEFAULT NULL,
+  `ZipCode` varchar(5) DEFAULT NULL,
+  `ZipCode_safe` varchar(10) DEFAULT NULL,
+  `Country_Code` varchar(255) DEFAULT NULL,
+  `International` tinyint(1) DEFAULT NULL,
+  `Latitude` varchar(255) DEFAULT NULL,
+  `Longitude` varchar(255) DEFAULT NULL,
+  `Pharmacy_Phone` varchar(255) DEFAULT NULL,
+  `Pharmacy_Phone_safe` varchar(255) DEFAULT NULL,
+  `Pharmacy_Fax` varchar(255) DEFAULT NULL,
+  `Types` varchar(255) DEFAULT NULL,
+  `Script_Msg_Accepted` varchar(255) DEFAULT NULL,
+  `Specialized_Msg_Accepted` varchar(255) DEFAULT NULL,
+  `Connectivity_Status` varchar(255) DEFAULT NULL,
+  `Accept_TSO` varchar(255) DEFAULT NULL,
+  `DEA_Audit_Exp` varchar(255) DEFAULT NULL,
+  `Test_Pharmacy` varchar(5) DEFAULT NULL,
+  `State_Wide_Mail_Order` varchar(6) NOT NULL,
+  `Created` datetime DEFAULT NULL,
+  `Modified` datetime DEFAULT NULL,
+  `Deleted` datetime DEFAULT NULL,
+  `24HR` varchar(3) DEFAULT NULL,
+  `on_weno` tinytext NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `id` (`id`),
+  UNIQUE KEY `ncpdp` (`NCPDP`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+#EndIf
+
+#IfNotTable weno_assigned_pharmacy
+CREATE TABLE IF NOT EXISTS `weno_assigned_pharmacy` (
+    `id` INT(10) NOT NULL AUTO_INCREMENT,
+    `pid` BIGINT(20) NOT NULL,
+    `primary_ncpdp` VARCHAR(8) NOT NULL,
+    `alternate_ncpdp` VARCHAR(8) NOT NULL,
+    PRIMARY KEY(id)
```

Comment:
would rec considering putting a key on pid column (not unique)

Reply:
would you suggest foreign key or index?